src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -129,6 +130,13 @@
             ])
             ->tag('console.command')
 
+        ->set('console.command.dotenv_debug', DotenvDebugCommand::class)
+            ->args([
+                param('kernel.environment'),
+                param('kernel.project_dir'),
+            ])
+            ->tag('console.command', ['is_enabled' => null])
```

Comment:
```suggestion
            ->tag('console.command', ['is_enabled' => !class_exists(Dotenv::class) ? false : null])
```

If the Dotenv component is not installed, we can disable the command right away.

Reply:
doesnt it override the logic at the code level? Currently we disable if dotenv _hasn't ran_, subtle.

Im not sure about  is_enabled either, tend to prefer staticness at the code level :shrug:

Reply:
> doesnt it override the logic at the code level?

Yes, that's the point.

> Currently we disable if dotenv _hasn't ran_, subtle.

All my change does is disabling the command if dotenv won't ever get the chance to run (because the component is missing).

Reply:
but it's different behavior :S

Reply:
Is it?

Reply:
OH. I missed your ternary :D :+1:

Reply:
> the null is rather pesky IMHO

I agree. 😞